store/sqlstore/team_store.go

```diff
@@ -470,15 +470,15 @@ func (s SqlTeamStore) PermanentDelete(teamId string) store.StoreChannel {
 	})
 }
 
-func (s SqlTeamStore) AnalyticsTeamCount() store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		c, err := s.GetReplica().SelectInt("SELECT COUNT(*) FROM Teams WHERE DeleteAt = 0", map[string]interface{}{})
-		if err != nil {
-			result.Err = model.NewAppError("SqlTeamStore.AnalyticsTeamCount", "store.sql_team.analytics_team_count.app_error", nil, err.Error(), http.StatusInternalServerError)
-			return
-		}
-		result.Data = c
-	})
+func (s SqlTeamStore) AnalyticsTeamCount() (int64, *model.AppError) {
+	c, err := s.GetReplica().SelectInt("SELECT COUNT(*) FROM Teams WHERE DeleteAt = 0", map[string]interface{}{})
+
+	if err != nil {
+		err := model.NewAppError("SqlTeamStore.AnalyticsTeamCount", "store.sql_team.analytics_team_count.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
You are shadowing the `err` variable here, which is something govet is going to complain for, besides, we normally don't create the variable to store the error, we just return it.

Reply:
Good catch!